### PR TITLE
Disclaimer command

### PR DESCRIPTION
**Pull request by jkiller295**
## Description

It's a fun command to give a disclaimer message in Vietnamese and English 
## Related Issue
[Issue 105](https://github.com/viet-aus-it/vait-discord-bot/issues/105)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/commands/disclaimer/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  );
+
+const DISCLAIMER_VI =
+  'Trong trường hợp nhóm này bị điều tra bởi các cơ quan trực thuộc bộ công an (hoặc các tổ chức chính trị tương tự), tôi khẳng định mình không liên quan tới nhóm hoặc những cá nhân khác trong nhóm này. Tôi không rõ tại sao mình lại có mặt ở đây, có lẽ tài khoản của tôi đã được thêm bởi một bên thứ ba. Tôi cũng xin khẳng định rằng mình không hề giúp sức cho những hành động của các thành viên trong nhóm này.' as const;
+const DISCLAIMER_EN =
+  'In case of an investigation by any federal entity or similar, I do not have any involvement with this group or with the people in it. I do not know how I am here, probably added by a third party, and I do not support any actions by the members of this group.' as const;
+
+export const getDisclaimer = async (interaction: CommandInteraction) => {
+  const lang = interaction.options.getString('lang');
+  const content = lang?.toLowerCase() === 'en' ? DISCLAIMER_EN : DISCLAIMER_VI;
```
Comment (dohaicuong) [inline src/commands/disclaimer/index.ts:19]: what if we go for `jap`

File: src/commands/disclaimer/index.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { CommandInteraction, MessageEmbed } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { Command } from '../command';
+
+const data = new SlashCommandBuilder()
+  .setName('disclaimer')
+  .setDescription('Get the VAIT disclaimer')
+  .addStringOption((option) =>
+    option.setName('lang').setDescription('Use en for English version')
```
Comment (dohaicuong) [inline src/commands/disclaimer/index.ts:9]: anh set dc suggestion `en`, `vi` luon day

File: src/commands/disclaimer/index.ts (hunk 1 above)
Comment (jkiller295) [reply src/commands/disclaimer/index.ts:19]: > what if we go for `jap`

give me `Jav` translation ;)

File: src/commands/disclaimer/index.ts (hunk 2 above)
Comment (jkiller295) [reply src/commands/disclaimer/index.ts:9]: > anh set dc suggestion `en`, `vi` luon day

teach me senpai

Comment (dohaicuong) [reply src/commands/disclaimer/index.ts:9]: https://discordjs.guide/interactions/slash-commands.html#choices

**Review by dohaicuong — APPROVED** [review]
lgtm

**Review by samhwang — APPROVED** [review]

**Pull request merged**